src/screens/PoolTogether/PoolTogetherWithdrawConfirm.js

```diff
@@ -0,0 +1,229 @@
+import * as React from 'react';
+import { RefreshControl, Platform } from 'react-native';
+import { connect } from 'react-redux';
+import styled, { withTheme } from 'styled-components/native';
+import { createStructuredSelector } from 'reselect';
+import type { NavigationScreenProp } from 'react-navigation';
+
+// actions
+import { logScreenViewAction } from 'actions/analyticsActions';
+import { fetchPoolPrizeInfo } from 'actions/poolTogetherActions';
+
+// constants
+import { SEND_TOKEN_PIN_CONFIRM } from 'constants/navigationConstants';
+import { POOLTOGETHER_WITHDRAW_TRANSACTION } from 'constants/poolTogetherConstants';
+
+// components
+import { ScrollWrapper } from 'components/Layout';
+import ContainerWithHeader from 'components/Layout/ContainerWithHeader';
+import { BaseText } from 'components/Typography';
+import Button from 'components/Button';
+
+// models
+import type { Dispatch, RootReducerState } from 'reducers/rootReducer';
+import type { Theme } from 'models/Theme';
+
+// selectors
+import { accountHistorySelector } from 'selectors/history';
+
+// utils
+import { themedColors } from 'utils/themes';
+import { fontStyles } from 'utils/variables';
+
+// local components
+import PoolTogetherWithdrawScheme from './PoolTogetherWithdrawScheme';
+
+
+const ContentWrapper = styled.View`
+  padding-top: ${Platform.select({
+    ios: '25px',
+    android: '19px',
+  })};
+  flex: 1;
+  justify-content: center;
+`;
+
+const Text = styled(BaseText)`
+  ${({ label }) => label ? fontStyles.regular : fontStyles.large};
+  letter-spacing: 0.18px;
+  color: ${({ label }) => label ? themedColors.secondaryText : themedColors.text};
+`;
+
+const ContentRow = styled.View`
+  flex-direction: row;
+  justify-content: center;
+  padding: 8px 20px 8px 20px;
+`;
+
+type Props = {
+  name: string,
+  navigation: NavigationScreenProp<*>,
+  session: Object,
+  logScreenView: (view: string, screen: string) => void,
+  fetchPoolStats: (symbol: string) => void,
+  theme: Theme,
+  user: Object,
+};
+
+type State = {
+  poolToken: string,
+  tokenValue: number,
+  tokenValueInFiat: string,
+  transactionPayload: Object,
+  feeInFiat: string,
+  feeDisplayValue: string,
+  isDisabled: boolean,
+};
+
+const poolTogetherLogo = require('assets/images/pool_together.png');
+
+class PoolTogetherWithdrawConfirm extends React.Component<Props, State> {
+  isComponentMounted: boolean = false;
+  scroll: Object;
+
+  constructor(props) {
+    const { navigation } = props;
+    const {
+      poolToken,
+      tokenValue,
+      tokenValueInFiat,
+      transactionPayload,
+      feeInFiat,
+      feeDisplayValue,
+      isDisabled,
+    } = navigation.state.params || {};
+    super(props);
+    this.state = {
+      poolToken,
+      tokenValue,
+      tokenValueInFiat,
+      transactionPayload,
+      feeInFiat,
+      feeDisplayValue,
+      isDisabled,
+    };
+  }
+
+  componentDidMount() {
+    const { logScreenView } = this.props;
+    this.isComponentMounted = true;
```

Comment:
Why is this even needed?

Reply:
removed extra property, kept the screen view logs